docs/pages/server-access/getting-started.mdx

```diff
@@ -115,49 +94,30 @@ This guide introduces some of these common scenarios and how to interact with Te
      </TabItem>
    </Tabs>
 
-3. Configure Teleport on the *Bastion Host*.
-
-   Teleport will now automatically acquire an X.509 certificate using the ACME protocol.
+   Next, we'll create a **join token** to add and start Teleport Server Access on the node.
 
-   ```code
-   # Configure Teleport with TLS certs
-   $ sudo teleport configure \
-     --acme --acme-email=your_email@example.com \
-     --cluster-name=tele.example.com \
-     -o file
-   ```
+## Step 2/4. Add a node to the cluster
 
-   Run the command above on `tele.example.com`.
-
-4. Launch your *Bastion Host* by running:
-
-   ```code
-   # Launch Teleport
-   $ sudo teleport start
-   ```
-
-   Next, we'll create a *join token* to add and start the second Node.
-
-## Step 2/4. Add a Node to the cluster
-
-1. Create a *join token* to add the second Node to the `tele.example.com` cluster. In a *Bastion Host* terminal run the following command:
+1. Create a join token to add the node to your Teleport cluster. In run the following command, either on your Auth Service host (for self-hosted deployments) or on your local machine (for Teleport Cloud).
 
    ```code
    # Let's save the token to a file
    $ sudo tctl tokens add --type=node | grep -oP '(?<=token:\s).*' > token.file
```

Comment:
Is sudo necessary for cloud users?

Reply:
I've suggested adding a `Details` box explaining tctl usage (including sudo) for Cloud users. If that looks good, we can look into making this an include and adding it to other relevant guides.